thirty-five-mm/app/models/category.rb

```diff
@@ -0,0 +1,32 @@
+class Category < ActiveRecord::Base
+  has_many :films
+
+  validates :name, presence: true, uniqueness: true
+
+  def category_count
```

Comment:
This shouldn't be an instance method - it's reading the whole collection. In any case you don't need this method - you can just call Category.count wherever you need it.